static/js/services/changes.js

```diff
@@ -41,36 +56,44 @@ angular.module('inboxServices').factory('Changes',
       });
     };
 
-    var RETRY_MILLIS = 5000;
-
-    var watchChanges = function() {
+    var watchChanges = function(meta) {
       $log.info('Initiating changes watch');
-      DB()
+      DB({ meta: meta })
         .changes({
           live: true,
-          since: lastSeq,
+          since: meta ? dbs.meta.lastSeq : dbs.medic.lastSeq,
           timeout: false,
           include_docs: true
         })
-        .on('change', notifyAll)
+        .on('change', function(change) {
+          notifyAll(meta, change);
+        })
         .on('error', function(err) {
           $log.error('Error watching for db changes', err);
           $log.error('Attempting changes reconnection in ' + (RETRY_MILLIS / 1000) + ' seconds');
-          $timeout(watchChanges, RETRY_MILLIS);
+          $timeout(function() {
+            watchChanges(meta);
+          }, RETRY_MILLIS);
         });
     };
 
     var init = function() {
       $log.info('Initiating changes service');
-      return DB().info().then(function(info) {
-        lastSeq = info.update_seq;
-      })
-      .then(watchChanges)
-      .catch(function(err) {
-        $log.error('Error initialising watching for db changes', err);
-        $log.error('Attempting changes initialisation in ' + (RETRY_MILLIS / 1000) + ' seconds');
-        return $timeout(init, RETRY_MILLIS);
-      });
+      return $q.all([
+        DB().info(),
+        DB({ meta: true }).info()
+      ])
```

Comment:
FWIW I think this can be an object not an array and then it binds cleaner:
```js
return $q.all({
  medic: DB().info(),
  meta: DB({meta: true}).info()
}).then(function(r) {
  dbs.medic.lastSeq = r.medic.update_seq;
  dbs.meta.lastSeq = r.meta.update_seq;
```